src/Aquifer.API/Endpoints/Notifications/List/Endpoint.cs

```diff
@@ -151,82 +161,101 @@ private static async Task<IReadOnlyList<ResourceCommentNotificationData>> GetRes
+                         pr.DisplayName AS ParentResourceDisplayName
+                     FROM Comments c
+                         JOIN Users u ON u.Id = c.UserId
+                         JOIN CommentThreads ct ON ct.Id = c.ThreadId
+                         JOIN ResourceContentVersionCommentThreads rcvct ON rcvct.CommentThreadId = ct.Id
+                         JOIN ResourceContentVersions rcv ON rcv.Id = rcvct.ResourceContentVersionId
+                         JOIN ResourceContents rc ON rc.Id = rcv.ResourceContentId
+                         JOIN Resources r ON r.Id = rc.ResourceId
+                         JOIN ParentResources pr ON pr.Id = r.ParentResourceId
+                     WHERE
+                         (
+                             EXISTS
+                             (
+                                 SELECT NULL
+                                 FROM
+                                 (
+                                     SELECT rcvauh.AssignedUserId,
+                                     ROW_NUMBER() OVER
+                                     (
+                                         PARTITION BY rcvauh.ResourceContentVersionId
+                                         ORDER BY rcvauh.Created DESC
+                                     ) AS Rank
+                                     FROM ResourceContentVersionAssignedUserHistory rcvauh
+                                     WHERE
+                                         rcvauh.ResourceContentVersionId = rcv.Id AND
+                                         rcvauh.Created < c.Created
+                                 ) mostRecentHistory
+                                 WHERE
+                                     mostRecentHistory.Rank = 1 AND
+                                     rcv.IsDraft = 1 AND
+                                     mostRecentHistory.AssignedUserId = @userId
+                             ) {(isContainingCommunityReviewerComments ? communityReviewerCommentsSubquery : "")}
+                              OR 
+                             EXISTS
+                             (
+                                 SELECT NULL
+                                 FROM CommentThreads ct2
+                                 JOIN Comments c2 ON c2.ThreadId = ct2.Id
+                                 WHERE
+                                     c.ThreadId = ct2.Id AND
+                                     c2.UserId = @userId AND
+                                     rcv.IsDraft = 1 AND
+                                     c.Created > c2.Created
+                             ) OR
+                             EXISTS
+                             (
+                                 SELECT NULL
+                                 FROM CommentMentions cm
+                                 WHERE
+                                     cm.CommentId = c.Id AND
+                                     cm.UserId <> c.UserId AND
+                                     rcv.IsDraft = 1 AND
```

Comment:
Why did the `IsDraft` check need to move?  My understanding is the comments all disappear after publishing so even community reviewer comments would disappear in the UI.

Reply:
Yes, it should be.